app/assets/javascripts/pages/vacancies/new/new.coffee

```diff
@@ -0,0 +1,14 @@
+App.Vacancies.New =
+  bind_fields: ->
+    $('#new_vacancy input').keypress (e) ->
```

Comment:
yeah! or even have default constructors, i dont remember if js has it right now, long time not doing frontend :(